tiles-generation/cdk/README.md

```diff
@@ -0,0 +1,59 @@
+# Tiles generation pipeline with CDK
+
+ This project in `tilesGenerationPipeline` folder helps to build a raster map tiles generaion pipeline on Amazon ECS in AWS CDK. The docker base image is come from https://github.com/Overv/openstreetmap-tile-server.
+
+## Diagram
+![alt text](./tiles-generation-diagram.png).
+
+#  Getting Started
+## Prerequisites
+
+### Install Node.js
+Visit [here](https://nodejs.org/en/) to install Node.js
+
+### Install AWS CDK Toolkit
+```
+npm install -g aws-cdk
+```
+
+### Install AWS CLI
+Visit [here](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) to install AWS CLI
+### AWS Prerequisites
+Visit [here](https://docs.aws.amazon.com/cdk/latest/guide/getting_started.html#getting_started_prerequisites) to configure aws credentials. 
+## Deployment
+
+### Deploy steps
+
+1. (Optional)The tiles generation configuration could be customized at `cdk.ts`. For example, Users can customize tiles generation area, zoom level, schedule generation, etc.
+
+
+2. A S3 bucket is required in CDK deployment to store generated tiles. The context name `TILE_S3_BUCKET` you will use to deploy. This bucket must be in the same aws account.
```

Comment:
Does S3 bucket need to be created before running the CDK deployment?

Reply:
Yes, currently we need an existing S3 bucket for deployment. In the future, we can support create new S3 bucket directly by CDK and use it.